lib/rubocop/cop/graphql/unused_argument.rb

```diff
@@ -119,6 +128,19 @@ def arg_end(node)
           node.loc.expression.end
         end
 
+        def inferred_arg_name(name_as_string)
+          case name_as_string
+          when /_id$/
+            name_as_string.sub(/_id$/, "").to_sym
+          when /_ids$/
+            name_as_string.sub(/_ids$/, "")
+                          .sub(/([^s])$/, "\\1s")
+                          .to_sym
+          else
+            name
+          end
+        end
```

Comment:
This was copypasted from https://github.com/rmosolgo/graphql-ruby/blob/master/lib/graphql/schema/member/has_arguments.rb#L25-L34. Not sure if some `NOTE: ...` comment required here